packages/pipeline/src/scripts/pull_radar_relay_orders.ts

```diff
@@ -0,0 +1,53 @@
+// tslint:disable:no-console
+import { HttpClient } from '@0x/connect';
+import * as R from 'ramda';
+import 'reflect-metadata';
+import { Connection, ConnectionOptions, createConnection, EntityManager } from 'typeorm';
+
+import { createObservedTimestampForOrder, SraOrder } from '../entities';
+import * as ormConfig from '../ormconfig';
+import { parseSraOrders } from '../parsers/sra_orders';
+import { handleError } from '../utils';
+
+const RADAR_RELAY_URL = 'https://api.radarrelay.com/0x/v2';
+const ORDERS_PER_PAGE = 10000; // Number of orders to get per request.
+
+let connection: Connection;
+
+(async () => {
+    connection = await createConnection(ormConfig as ConnectionOptions);
+    await getOrderbook();
```

Comment:
```suggestion
    await getOrderbookAsync();
```

Reply:
@fabioberger Fixed, but I'm really confused about why the linter didn't catch this one. Any ideas?

Reply:
Hm... same. Just took a look at `async_suffix.ts` and looks like we only check `ts.MethodDeclaration` and not `ts.FunctionDeclaration`. Gonna add a todo to fix this to the backlog.